src/NUnitEngine/nunit.engine/Services/TestAgency.cs

```diff
@@ -186,7 +186,7 @@ private Guid LaunchAgentProcess(TestPackage package)
             if (debugAgent)
                 agentArgs += " --debug-agent";
             if (workDirectory != string.Empty)
-                agentArgs += " --work=" + workDirectory;
+                agentArgs += " --work \"" + workDirectory + "\"";
```

Comment:
Thanks for this @Funbit 
I don't know much about this part of the code, but how will this behave if someone tries to pass in a path already wrapped in double quotes?